mintpy/objects/stack.py

```diff
@@ -506,6 +508,48 @@ def get_design_matrix4step_func(date_list, step_date_list):
 
             return A
 
+
+        def get_design_matrix4exp_func(date_list, exp_list):
+            """design matrix/function model of exponential postseismic relaxation estimation
+            Parameters: date_list      : list of dates in YYYYMMDD format
+                        exp_list[0]    : Exp func(s) onset time(s) with date in YYYYMMDD
+                        exp_list[1]    : tau(s), characteristic time(s) in decimal days
+            Returns:    A              : 2D array of zeros & ones in size of (num_date, num_exp)
+            """
+            num_date = len(date_list)
+            num_exp  = len(exp_list[0])
+            A = np.zeros((num_date, num_exp), dtype=np.float32)
+
+            t = np.array(ptime.yyyymmdd2years(date_list))
+            t_exps = ptime.yyyymmdd2years(exp_list[0])
+            tau_exps = np.array(exp_list[1]) / 365.25
+            for i, (t_exp, tau_exp) in enumerate(zip(t_exps, tau_exps)):
+                A[:, i] = np.array(t > t_exp).flatten() * (1-np.exp(-(t-t_exp)/(tau_exp)))
```

Comment:
The `np.exp(-(t-t_exp))` part is different from the equation (4) of Hetland et al. (2012), and I think yours is correct. Could you put a comment here on a correct reference?

Considering the various forms of exp equations out there, as Mark pointed out, it would be useful to write the equations implemented here explicitly in the function comment above as well, such as [here](https://github.com/insarlab/MintPy/blob/main/mintpy/timeseries2velocity.py#L515) and [here](https://github.com/insarlab/MintPy/blob/main/mintpy/utils/solvers/l1regls.py#L25).

Same comment for the log func as well.

Reply:
Yeap, I will put the equation explicitly in the comment. Also for the log func.

Reply:
I have added the explicit equation in the comment. I still reference Hetland et. al., 2012. 

That seems to be simply a typo in Hetland et. al. (2012). Because in his MInTS codes, he actually wrote `np.exp(-(t-onset_time))`.

I read references that Hetland cited,  e.g., Savage et al., 2003, 2005, also Hetland and Hager, 2006. But nothing is as clear as Hetland et. al. (2012).

Hence, in the comment part of the MintPy code, I explain that there is a typo in Hetland et. al. (2012), and the variable inside the exp (and the log) term should be *the time after the earthquake*, i.e., `(t - onset_time)`

Reply:
Great, that is very clear @yuankailiu.